observability/logging/logging_release_notes/logging-5-8-release-notes.adoc

```diff
@@ -10,6 +10,8 @@ include::snippets/logging-compatibility-snip.adoc[]
 
 include::snippets/logging-stable-updates-snip.adoc[]
 
+include::modules/logging-release-notes-5-8-15.adoc[leveloffset=+1]
+
 include::modules/logging-release-notes-5-8-14.adoc[leveloffset=+1]
```

Comment:
The preview for this module says: "This release includes [OpenShift Logging Bug Fix Release 5.8.14](https://access.redhat.com/errata/RHSA-2024:8317) and [OpenShift Logging Bug Fix Release 5.8.14](https://access.redhat.com/errata/RHBA-2024:8316)." Should one of those be release 5.8.15 or . . . .?

Reply:
No, one is for kibana and the other for the CLO, the links go to different errata.